server/index.js

```diff
@@ -5,32 +5,89 @@ const port = 3000;
 const bodyParser = require('body-parser');
 const axios = require('axios');
 const config = require('../config');
+const isaacAPI = require('./QuestionAnswerAPI');
+const louisAPI = require('./ProductOverviewAPI');
+const helenaAPI = require('./RatingsReviewsAPI');
 
-app.use(express.static('client'));
+app.use(express.static(`${__dirname} /../client/dist`));
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: true }));
 
 // pass in string to get from that endpoint
 const getAPI = (params, cb) => {
-  axios({
+  const options = {
     method: 'GET',
-    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/${params}`,
+    url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/${params}?count=100`,
     headers: { Authorization: config.gitToken },
-  }).then((response) => {
-    cb(response);
-  });
+  };
+  axios(options)
+    .then((data) => {
+      console.log('axios get success');
+      return cb(data);
+    })
+    .catch(() => {
+      console.log('catch axois error');
+    });
 };
 
 app.get('/', (req, res) => {
-  getAPI('products', (err, results) => {
+  res.send('success');
+});
+
+app.post('/', (req, res) => {
+  this.getAPI(`products`, (err, results) => {
```

Comment:
Just curious as to why we have to use "this.getAPI" rather than just invoking "getAPI"?

Reply:
Seems we don't need this ,but works fine either way. Can edit that out.

Reply:
Oh ok, we can take it out on the next pull request or something then.